non-oss/pharos_pro/addons/kontena-lens/resources/07-dashboard-deployment.yml.erb

```diff
@@ -32,6 +32,8 @@ spec:
               value: "<%=  user_management %>"
             - name: CHARTS_ENABLED
               value: "<%=  charts_enabled %>"
+            - name: KUBE_METRICS
```

Comment:
Please change `KUBE_METRICS` to `KUBE_METRICS_URL`

Reply:
Fixed.